src/config/wallets.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { ReactComponent as PolkadotExtensionLogoIcon } from 'assets/img/polkadot-extension-logo.svg';
+import { ReactComponent as TalismanWalletLogoIcon } from 'assets/img/talisman-wallet-logo.svg';
+import { ReactComponent as SubWalletLogoIcon } from 'assets/img/subwallet-logo.svg';
+
+enum WalletSourceName {
+  PolkadotExtensionLogoIcon = 'polkadot-js',
+  Talisman = 'talisman',
+  SubWallet = 'subwallet-js'
+}
+
+interface WalletData {
+  name: string;
+  LogoIcon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+  URL: string;
```

Comment:
I wonder if name and URL fields are cased consistently. name & url or NAME & URL